File: Ansh/Array/python/java/LinearSearch.java
class LinearSearch{
    public int LinearSearch(int[] nums,int target){
        
        for(int i=0;i<nums.length;i++){
            if (nums[i]==target){return i;}
            
        }
        return -1;

    }
    public static void main(String[]args){
        int[]nums={1,2,3,4,5,6};
        int target=5;
        LinearSearch sol=new LinearSearch();
        int result=sol.LinearSearch(nums, target);
        System.out.println(result);
        System.out.println("Hello");
    }

}

File: Ansh/Array/python/LinearSearch.py
class Solution:
    def linearSearch(self, nums, target):
        if target not in nums:
            return -1
        else:
            for i in range(len(nums)):
                if nums[i]==target:
                    return i
        
            
if __name__=="__main__":
    s=Solution()
    print(s.linearSearch([1,2,3,4,5,6],6))
    
